src/frontends/paddle/docs/operation_mapping_flow.md

```diff
@@ -0,0 +1,15 @@
+# OpenVINO Paddle Frontend operator enabling Flow
+
+1. Declare `CreatorFunction` for the Paddle operator and register it to the map in `src/op_table.cpp`.
+   * The map is retrived from:
+     ```cpp
+     std::map<std::string, CreatorFunction> get_supported_ops() {
```

Comment:
I think will be better to add a real code to such place, please take a look: https://github.com/openvinotoolkit/openvino/blob/master/src/frontends/ir/docs/support_new_opset.md
Here I insert a part of the existed file in the md document.

Reply:
done